README.md

```diff
@@ -55,3 +109,15 @@ From within the root directory in a active virtual environment run:
 pip install .
 ```
 This will download the relevant dependencies into the venv as well as setting up the datasets that we will be using in the course.
+
```

Comment:
Do we need to add familiarity with virtual environments to the required previous knowledge?

Reply:
I'd say not, as we don't want to overload the quantity of pre-reading and we guide them through it quite carefully.